src/components/pages/AdminDashboard/DevToolsNew.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import { connect } from 'react-redux';
+
+import { Menu, Dropdown, Button } from 'antd';
+import { DownOutlined } from '@ant-design/icons';
+
+import { devMode } from '../../../state/actions/index';
+
+const DevToolsNew = ({ devMode, setDevMode }) => {
+  const { push } = useHistory();
+
+  const handleSatMon = e => {
+    push('/dev/day/1-3');
+  };
+
+  const handleTues = e => {
+    push('/dev/day/4');
+  };
+
+  const handleWed = e => {
+    push('/dev/day/5');
+  };
+
+  const handleThurs = e => {
+    push('/dev/day/6');
+  };
+
+  const handleFri = e => {
+    push('/dev/day/7');
+  };
+
+  const handleDevMode = () => {
+    setDevMode(!devMode.isDevModeActive);
+  };
+
+  const dropdown = (
+    <Menu>
+      <Menu.Item>
+        <a target="_blank" onClick={handleSatMon}>
+          Saturday - Monday
+        </a>
+      </Menu.Item>
+      <Menu.Item>
+        <a target="_blank" onClick={handleTues}>
+          Tuesday
+        </a>
+      </Menu.Item>
+      <Menu.Item>
+        <a target="_blank" onClick={handleWed}>
+          Wednesday
+        </a>
+      </Menu.Item>
+      <Menu.Item>
+        <a target="_blank" onClick={handleThurs}>
```

Comment:
This could possibly be converted to a react-router <Link tag with the "to" property, which could reduce the need for the handlers and would fix the current eslint issues (referring to a <a> without href)